edx_arch_experiments/summaryhook_aside/block.py

```diff
@@ -8,11 +8,14 @@
 from edx_arch_experiments.summaryhook_aside.waffle import summary_enabled, summary_staff_only
 
 summary_fragment = """
+<div>&nbsp;</div>
```

Comment:
is this supposed to be here?

Reply:
Good eye but actually yes! it's a spacer to keep the result box from rubbing against the staff buttons which annoys the UX guy, the first diff puts this in on purpose.